include/mujincontrollerclient/mujinjson.h

```diff
@@ -358,6 +358,7 @@ inline void LoadJsonValue(const rapidjson::Value& v, unsigned long long& t) {
     }
 }
 
+#ifndef _MSC_VER
```

Comment:
Should we `#include <stdint.h>` instead? Based on https://stackoverflow.com/questions/13604137/definition-of-int64-t

Reply:
@hemangandhi The thing is, there exists a branch which supports VC2010 https://github.com/mujin/controllerclientcpp/tree/savebackup and VC2010 does not support `<stdint.h>`.